velox/dwio/dwrf/test/ReaderTest.cpp

```diff
@@ -1737,6 +1737,40 @@ TEST(TestReader, fileColumnNamesReadAsLowerCaseComplexStruct) {
   EXPECT_EQ(col0_1_1_0_0->childByName("ccint3"), col0_1_1_0_0_0);
 }
 
+TEST(TestReader, testOrcCharType) {
+  // test_char.orc has schema: struct<d_date_id:char(16),type_id:char(1)>
+  auto pool = getDefaultPool().get();
+  dwio::common::ReaderOptions readerOpts{pool};
+  readerOpts.setFileFormat(dwio::common::FileFormat::ORC);
+
+  const std::shared_ptr<const RowType> expectedType =
+      std::dynamic_pointer_cast<const RowType>(HiveTypeParser().parse(
```

Comment:
Why not ROW({"d_date_id", "type_id"}, {VARCHAR(), VARCHAR()})?

Reply:
Fixed.